build.gradle

```diff
@@ -102,25 +102,31 @@ task loadData{
 
 task loadBundleResource{
     doLast {
-        def bundleFile = new File(projectDir, "bundlefile/PASExample.json")
```

Comment:
Is "bundlefile/PASExample.json" not needed anymore?

Reply:
Yes, it will be removed together with EchoCardiography rule set.